app/modules/projects/projects_service.py

```
@@ -289,9 +344,18 @@ def create_project(db: Session, project: Project):
         project.created_at = datetime.utcnow()
         project.updated_at = datetime.utcnow()
         db.add(project)
-        db.commit()
-        db.refresh(project)
-        return project
+        try:
+            db.commit()
+            db.refresh(project)
+            return project
```

Ruff (0.14.1)

Consider moving this statement to an `else` block

(TRY300)